### PR TITLE
Update perms for Greninja

### PR DESCRIPTION
**Pull request by ThisChicken**

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -1,4 +1,37 @@
+
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: remove blank line

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -1,4 +1,37 @@
...
+        permissions:
+        - -essentials.protect.damage.fall
+        - powers.wallrun   
+        - ch.alias.jump0
+        - ch.alias.jump1
+        - ch.alias.jump2
+        - ch.alias.jump3
+        - ch.alias.jump4
+        - ch.alias.jump5
+        - ch.alias.jump6
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: jump should be limited to 4 unless you have proof he can jump really high

File: PermissionsEx/permissions.yml [hunk 3]
```diff
@@ -1,4 +1,37 @@
...
+        - powers.wallrun   
+        - ch.alias.jump0
+        - ch.alias.jump1
+        - ch.alias.jump2
+        - ch.alias.jump3
+        - ch.alias.jump4
+        - ch.alias.jump5
+        - ch.alias.jump6        
+        - ch.alias.invis
+        - ch.alias.vis
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: requires proof of invisibility

File: PermissionsEx/permissions.yml (hunk 2 above)
Comment (ThisChicken) [inline PermissionsEx/permissions.yml]: http://static.fjcdn.com/gifs/Dcm_1f8681_5383610.gif

File: PermissionsEx/permissions.yml (hunk 3 above)
Comment (ThisChicken) [inline PermissionsEx/permissions.yml]: in the games, he can learn camouflage

File: PermissionsEx/permissions.yml (hunk 2 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: approved

File: PermissionsEx/permissions.yml (hunk 3 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: requires a video for proof

File: PermissionsEx/permissions.yml (hunk 2 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: approved

File: PermissionsEx/permissions.yml (hunk 3 above)
Comment (ThisChicken) [inline PermissionsEx/permissions.yml]: found him using shadow sneak, witch turns him invis https://www.youtube.com/watch?v=S0fHe9Yg-vk
at 1:22

**Pull request merged**